src/orchestration/polling_message_handler.go

```diff
@@ -2,19 +2,30 @@ package orchestration
 
 import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azqueue"
+	"github.com/CDCgov/reportstream-sftp-ingestion/sftp"
+	"github.com/CDCgov/reportstream-sftp-ingestion/utils"
+	"log/slog"
 )
 
 type PollingMessageHandler struct {
-	// TODO - add SFTP handler
-}
-
-func NewPollingMessageHandler() (PollingMessageHandler, error) {
-	// TODO - add SFTP handler
-
-	return PollingMessageHandler{}, nil
 }
 
 func (receiver PollingMessageHandler) HandleMessageContents(message azqueue.DequeuedMessage) error {
-	// TODO - use SFTP handler
+	slog.Info("Handling polling message", slog.String("message text", *message.MessageText))
+	// TODO - use the message contents to figure out stuff about config and files
+	// SFTP handler has hard-coded details about where to retrieve files from
+	sftpHandler, err := sftp.NewSftpHandler()
+	if err != nil {
+		slog.Error("ope, failed to create sftp handler", slog.Any(utils.ErrorKey, err))
```

Comment:
Did we want to change "ope"?

Reply:
ope, I guess I should clean up my midwesternisms